lib/perl/Genome/Model/ReferenceAlignment/Command/BsmapMethCalcConversionRate.pm

```diff
@@ -0,0 +1,100 @@
+package Genome::Model::ReferenceAlignment::Command::BsmapMethCalcConversionRate;
+
+use strict;
+use warnings;
+use Genome;
+use Genome::Model::Tools::Bsmap::MethCalcConversionRate;
+
+class Genome::Model::ReferenceAlignment::Command::BsmapMethCalcConversionRate {
+    is => 'Command::V2',
+    has => [
+        model => {
```

Comment:
Just to note... If this will be added to the pipeline, it will need to accept a specific build instead of a model.  (From the command line it could still be passed a model ID, and it will prompt the user to select one of the builds.)